src/org/zaproxy/zap/extension/ascanrulesBeta/resources/help/contents/ascanbeta.html

```diff
@@ -19,8 +19,12 @@
 SilverLight's clientaccesspolicy.xml. 
 
 <H2>CSRF Token</H2>
-Scans for the existence of anti-CSRF tokens. 
-Alerts on requests which do not appear to contain anti-CSRF tokens.
+Scans for the existence of anti-CSRF tokens. <br>
+Alerts on requests which do not appear to contain anti-CSRF tokens.<br>
```

Comment:
L22 & L23, should this be `Anti-CSRF` to be consistent with the info in messages.properties?